src/utils/staking/index.ts

```diff
@@ -2,7 +2,25 @@ import { networks, payments } from "bitcoinjs-lib";
 import { Taptree } from "bitcoinjs-lib/src/types";
 import { internalPubkey } from "../../constants/internalPubkey";
 import { PsbtOutputExtended } from "../../types/psbtOutputs";
+import { Params } from "../../types/params";
+import { StakingError, StakingErrorCode } from "../../error";
+import { UTXO } from "../../types/UTXO";
+import { isValidNoCoordPublicKey } from "../btc";
 
+// minimum unbonding output value to avoid the unbonding output value being
+// less than Bitcoin dust
+const MIN_UNBONDING_OUTPUT_VALUE = 1000;
```

Comment:
Is this `1000` a specific value defined by the protocol? I recall we have a constant defining bitcoin dust in this library, so I wonder whether this minimum value does not refer to bitcoin dust or something else.